kornia/testing/__init__.py

```diff
@@ -128,3 +130,44 @@ def _get_precision_by_name(
         return tol_val
 
     return tol_val_default
+
+
+try:
```

Comment:
@pmeier probably we should add here a note or todo saying that if any time we do not support <1.9 - we should directly use `torch.testing.assert_close` ?